docs/team/alekkwek18.md

```diff
@@ -5,22 +5,39 @@ title: Alek Kwek's Project Portfolio Page
 
 Given below are my contributions to the project.
 
-* **New Feature**: NIL
+* **New Feature**: Added `DeleteContactCommand`.
+  * **What it does**: Allows the user to delete a specific contact from the address book.
+  * **Justification**: This feature is essential for recruitment workflows, enabling users to manage and clean up their contact lists.
+  * **Highlights**: This command was refactored from `DeleteCommand` to `DeleteContactCommand`, with `DeleteCommand` made generic to support future deletions of other entity types, such as jobs and companies.
 
-* **Code contributed**: NIL
+* **New Feature**: Added `ScreenCommand`.
+  * **What it does**: Allows the user to filter and display contacts that match specific job requirements.
+  * **Justification**: This command enhances the recruiter’s efficiency by enabling faster matching of candidates to job listings.
+  * **Highlights**: Implementing this command involved defining filtering criteria that integrate seamlessly with existing job and contact entities.
+
+* **New Feature**: Added `UnmatchCommand`.
+  * **What it does**: Allows the user to remove associations between jobs and contacts.
+  * **Justification**: This feature improves data accuracy by allowing recruiters to update job matches as candidate-job statuses change.
+  * **Highlights**: The implementation required careful handling of relationship data, as it impacted associated entities and existing commands.
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2425s1.github.io/tp-dashboard/?search=alekkwek18&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&since=2024-09-20&tabOpen=true&tabType=authorship&tabAuthor=AlekKwek18&tabRepo=AY2425S1-CS2103-F13-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false&checkedFileTypes=docs~functional-code~test-code~other)
 
 * **Project management**:
-    * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+  * NIL
 
-* **Enhancements to existing features**: NIL
+* **Enhancements to existing features**:
+  * Refactored `AddCommand` to `AddContactCommand` to improve specificity and clarity.
+  * Refactored `DeleteCommand` into a generic command `DeleteCommand<T>` to support future implementations of `DeleteJob` and `DeleteCompany`.
 
 * **Documentation**:
-    * User Guide: NIL
-    * Developer Guide: NIL
+  * User Guide: NIL
+  * Developer Guide:
+    * Updated UML diagram to reflect changes in `DeleteContactCommand` and generic refactoring of `DeleteCommand`. (pull requests [#135](https://github.com/AY2425S1-CS2103-F13-4/tp/pull/135)) 
+
+* **Testing**:
```

Comment:
I dont think this is needed since we are expected to write test for new features anyways, but there is no instruction on portfolio so we could just keep if for now.